soh/src/code/sys_matrix.c

```diff
@@ -552,15 +552,15 @@ Mtx* Matrix_MtxFToMtx(MtxF* src, Mtx* dest) {
     return dest;
 }
 
-Mtx* Matrix_ToMtx(Mtx* dest) {
+Mtx* Matrix_ToMtx(Mtx* dest, char* file, s32 line) {
     FrameInterpolation_RecordMatrixToMtx(dest, __FILE__, __LINE__);
-    guMtxF2L(Matrix_CheckFloats(sCurrentMatrix), dest);
+    guMtxF2L(MATRIX_CHECKFLOATS(sCurrentMatrix), dest);
```

Comment:
You would not call the macro version of the function call here. Just pass through file and line function arguments.